module/parser/src/commonTest/kotlin/org/cru/godtools/tool/model/ButtonTest.kt

```diff
@@ -104,17 +104,31 @@ class ButtonTest : UsesResources() {
         }
     }
 
+    // region isIgnored
     @Test
-    fun testButtonTypeUnknown() {
-        val button = Button(Manifest(), type = Button.Type.UNKNOWN)
-        assertTrue(button.isIgnored)
+    fun testIsIgnoredClickable() {
+        with(Button()) {
+            assertFalse(isClickable)
+            assertTrue(isIgnored)
+        }
+
+        with(Button(events = listOf(EventId.FOLLOWUP))) {
+            assertTrue(isClickable)
+            assertFalse(isIgnored)
+        }
+
+        with(Button(url = TEST_URL)) {
+            assertTrue(isClickable)
+            assertFalse(isIgnored)
+        }
     }
 
     @Test
     fun testButtonStyleUnknown() {
-        val button = Button(Manifest(), style = Button.Style.UNKNOWN)
+        val button = Button(style = Button.Style.UNKNOWN, events = listOf(EventId.FOLLOWUP))
```

Comment:
I'm trying to make the connection between the changes in ```Button.kt``` and the removal of ```Manifest()``` here. Was it something related to ```type``` being removed in the test declaration?

Reply:
it was an optimization I had previously made to the test constructor for `Button`. the first parameter to the test constructor has a default value of `Manifest()` already, so it wasn't necessary to pass it in here.